src/lib/operators/import_binary.hpp

```diff
@@ -79,6 +84,46 @@ class ImportBinary : public AbstractReadOnlyOperator {
    */
   static std::shared_ptr<Chunk> _import_chunk(std::ifstream& file, std::shared_ptr<Table>& table);
 
+  /*
+   * Reads the header from the given file.
+   * Creates an empty table from the extracted information and
+   * returns that table and the number of chunks.
+   * The header has the following format:
```

Comment:
This looks like `export_binary.hpp` (and well... it should). I would refer to that comment, because otherwise someone else *will* forget to update one of the comments in the future.

Reply:
We did it like this to stay consistent with the existing comments. If this is changed, probably all such comments in `import_binary.hpp` should refer to comments in `export_binary.hpp`.